lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -543,44 +543,51 @@ internal class EvaluatingCompiler(
         }
 
     private fun compilePlus(expr: PartiqlAst.Expr.Plus, metas: MetaContainer): ThunkEnv {
+        if (expr.operands.size < 2) {
+            error("Internal Error: PartiqlAst.Expr.Plus must have at lest 2 arguments")
```

Comment:
typo: "... have at **least** 2 arguments"

```suggestion
            error("Internal Error: PartiqlAst.Expr.Plus must have at least 2 arguments")
```